src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngineUnitTest.java

```diff
@@ -254,6 +272,47 @@ public void testCoverageAnnotationOnDbSnpSite() throws Exception {
     }
 
     @Test
+    public void testCoverageAnnotationOnOverlapSite() throws Exception {
```

Comment:
In addition to this new test, please also add:

1. a test involving both a separate DBSNP input and a non-DBSNP input (different file!) passed in via the new List argument

2. a test with a DBSNP input passed in both individually and via the List, showing that we throw in such a case (assuming you take my suggestion above)